src/python/pants/core/goals/tailor_test.py

```diff
@@ -65,12 +66,30 @@ class FortranTests(Target):
     core_fields = (FortranTestsSources,)
 
 
+class FortranModule(Target):
+    alias = "fortran_module"
```

Comment:
This could use a comment explaining why it exists, i.e., what it serves to test.

Reply:
Added.